### PR TITLE
Added description and url to embed of xkcd command

### LINKED ISSUE #588: Improve xkcd command
## Description
The current xkcd command doesn’t have the description, or a link to the comic.

## Reasoning
It would help provide more info about the comic.

## Proposed Implementation
Use the ‘alt’ key in the json page and use the URL var we use to make the requests.

## Would you like to implement this yourself?

- [x] I'd like to implement this feature myself
- [ ] Anyone can implement this feature

### PR DESCRIPTION
**Pull request by MrKomodoDragon**
## Relevant Issues

Closes #588. 

## Description
I set the description the the alt key in the json page, and set a url in the title to the link to the comic.

## Reasoning
The `alt` key from the request was already available, and could be safely added to the embed. The URL was also available.

## Screenshots
**Before Changes :**
image

**After Changes: **
image

## Did you:

- [x] Join the [**Python Discord Community**](https://discord.gg/python)?
- [x] If dependencies have been added or updated, run `pipenv lock`?
- [x] **Lint your code** (`pipenv run lint`)?
- [x] Set the PR to **allow edits from contributors**?

### CODE REVIEW

File: bot/exts/evergreen/xkcd.py [hunk 1]
```diff
@@ -69,6 +69,8 @@ async def fetch_xkcd_comics(self, ctx: Context, comic: Optional[str]) -> None:
                     return
 
         embed.title = f"XKCD comic #{info['num']}"
+        embed.description = info['alt']
+        embed.url = f"{BASE_URL}/{comic}"
```
Comment (ChrisLovering) [inline bot/exts/evergreen/xkcd.py:73]: This won't work if the user does `.xkcd latest` as the `comic` var isn't updated to be the comic number.

**Review by ChrisLovering — CHANGES_REQUESTED** [review]

File: bot/exts/evergreen/xkcd.py (hunk 1 above)
Comment (MrKomodoDragon) [reply bot/exts/evergreen/xkcd.py:73]: ok

Comment (MrKomodoDragon) [reply bot/exts/evergreen/xkcd.py]: Fixed.

**Review by ChrisLovering — APPROVED** [review]
Looks good to me!

**Review by ks129 — APPROVED** [review]

**Pull request merged**